docker-compose.yml

```diff
@@ -48,16 +54,20 @@ services:
       - OPENFORMS_LOCATION_CLIENT=${OPENFORMS_LOCATION_CLIENT:-openforms.contrib.bag.client.BAGClient}
       - TWO_FACTOR_FORCE_OTP_ADMIN=0
       - TWO_FACTOR_PATCH_ADMIN=0
+      - CORS_ALLOW_ALL_ORIGINS=${CORS_ALLOW_ALL_ORIGINS:-false}
```

Comment:
Relates to #1366

Reply:
Should I leave it here?

Reply:
yeah, it's alright! I'll leave that one open as we may to switch the default value to true for local dev/testing